lib/controllers/addOn.js

```diff
@@ -1,67 +1,54 @@
-var modulus = require('../modulus'),
-    util = require('util'),
-    librarian = require('../common/api').librarian,
-    userConfig = require('../common/api').userConfig,
-    Errors = require('../util/errors'),
-    async = require('async');
+const Librarian = require('../common/api').librarian;
+const UserConfig = require('../common/api').userConfig;
+const Errors = require('../util/errors');
 
-//-----------------------------------------------------------------------------
-var AddOn = function() {
+// ----------------------------------------------------------------------------
+var AddOn = function () {
 
 };
 
-//-----------------------------------------------------------------------------
+// ----------------------------------------------------------------------------
 AddOn.prototype.getById = function (id, callback) {
-  librarian.addOns.getById(id, function (err, addon) {
-    if (err) {
-      return callback(Errors.getMessage(err));
-    }
-
-    callback(null, addon);
-  });
+  Librarian.addOns.getById(
+    id, function (err, addon) {

```

Comment:
^ Probably for consistency with the rest of the long line calls below. It's ugly, but the real culprit here is `librarian` exposing functions that take too many parameters.